developers/weaviate/modules/retriever-vectorizer-modules/text2vec-jinaai.md

```diff
@@ -0,0 +1,161 @@
+     - "8080:8080"
+    environment:
+      QUERY_DEFAULTS_LIMIT: 20
+      AUTHENTICATION_ANONYMOUS_ACCESS_ENABLED: 'true'
+      PERSISTENCE_DATA_PATH: "./data"
+      # highlight-start
+      ENABLE_MODULES: text2vec-jinaai
+      DEFAULT_VECTORIZER_MODULE: text2vec-jinaai
+      JINAAI_APIKEY: sk-foobar  # Setting this parameter is optional, you can
+                                #   also provide the key at query time.
+      # highlight-end
+      CLUSTER_HOSTNAME: 'node1'
+...
+```
+
+## Collection configuration
+
+To configure how the module behaves in each collection, update the [Weaviate schema](/developers/weaviate/configuration/schema-configuration.md).
+
+### API settings
+
+#### Parameters
+
+|Parameter|Required|Default|Purpose|
+|:-|:-|:-|:-|
+|`model`|No|`jina-embeddings-v2-base-en`|A model name, e.g. `jina-embeddings-v2-small-en`.|
+
+#### Example
+
+The following example configures the `Document` collection by setting the vectorizer to `text2vec-jinaai` and the model to `jina-embeddings-v2-small-en`:
+
+```json
+{
+  "classes": [
+    {
+      "class": "Document",
+      "description": "A collection called document",
+      // highlight-start
+      "vectorizer": "text2vec-jinaai",
+      "moduleConfig": {
+        "text2vec-jinaai": {
+          "model": "jina-embeddings-v2-small-en",
+        }
+      },
+      // highlight-end
+    }
+  ]
+}
+```
+
+
+### Vectorization settings
+
+You can set vectorizer behavior using the `moduleConfig` section under each collection and property:
+
+#### Collection level settings
+
+|Parameter|Default|Purpose|
+|:-|:-|:-|
+|`vectorizer`|-| Use this module to vectorize the data.|
+|`vectorizeClassName`| `true`| When `true`, vectorizes the class name.
+
+#### Property level settings
+
+|Parameter|Default|Purpose|
+|:-|:-|:-|
+|`skip`|`false`|When `true`, does not vectorize the property.|
+|`vectorizePropertyName`|`true`|When `true`, vectorizes the property name.
+
+#### Example
+
+```json
+{
+  "classes": [
+    {
+      "class": "Document",
+      "description": "A collection called document",
+      "vectorizer": "text2vec-jinaai",
+      "moduleConfig": {
+        "text2vec-jinaai": {
+          "model": "jina-embeddings-v2-small-en",
+          // highlight-start
+          "vectorizeClassName": 'false'
+          // highlight-end
+        }
+      },
+      "properties": [
+        {
+          "name": "content",
+          "dataType": ["text"],
+          "description": "Content that will be vectorized",
+          // highlight-start
+          "moduleConfig": {
+            "text2vec-jinaai": {
+              "skip": 'false',
+              "vectorizePropertyName": 'false'
+            }
+          }
+          // highlight-end
+        }
+      ]
+    }
+  ]
+}
+```
+
+## Query-time parameters
+
+### API key
+
+You can supply the API key at query time by adding it to the HTTP header.
+
+|HTTP Header|Value|Purpose|
+|:-|:-|:-|
+|`X-Jinaai-Api-Key`|`YOUR-JINAAI-API-KEY`|JinaAI key|
```

Comment:
Let's add a `Usage example` section like we have here:

https://weaviate.io/developers/weaviate/modules/retriever-vectorizer-modules/text2vec-openai#usage-example